bumpversion/cli.py

```diff
@@ -66,6 +67,42 @@
 ]
 
 
+def main(original_args=None):
```

Comment:
Perhaps this function could some newlines in between to separate parts of the functionality. Surely there's some init phase, parser setup phase, and runtime phase we can separate.

Reply:
agree, feels quite dense, maybe some newlines and if you want: comments